app/scripts/modules/core/pipeline/config/stages/bake/aws/bakeStage.html

```diff
@@ -71,6 +71,41 @@ <h4 ng-if="viewState.loading" class="text-center">
         <input type="text" class="form-control input-sm"
                ng-model="stage.templateFileName"/>
       </stage-config-field>
+      <stage-config-field label="Extended Attributes" help-key="pipeline.config.bake.extendedAttributes"
+                          ng-if="viewState.roscoMode || stage.templateFileName">
```

Comment:
I think should show the extended attributes if either `roscoMode` is enabled, or the `extendedAttributes` map has at least one entry. It doesn't have to be tied to the presence of `templateFileName` (I think this is likely copy/paste error).